lxd/container_lxc.go

```diff
@@ -1371,6 +1373,15 @@ func (c *containerLXC) Start(stateful bool) error {
 		return err
 	}
 
+	ctxMap = log.Ctx{"name": c.name,
+		"action":        op.action,
+		"creation date": c.creationDate,
+		"ephemeral":     c.ephemeral,
+		"last used":     c.lastUsedDate,
```

Comment:
I think we should stay away from spaces in the keys as that makes it pretty difficult to read in the log